src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container containerBig">
+      <form
+        action="#"
+        method="get"
+        data-qa="big"
+      >
+        <label for="bigSearch">
+          <img
+            src="./images/Search.svg"
+            class="bigIcon"
+            alt="icon"
+          >
+        </label>
+        <input
+            id="bigSearch"
+            type="search"
+            class="search bigSearch fokyc"
+            data-qa="keypress"
+            placeholder="Try &quot;Los Angeles&quot;"
+          >
+      </form>
+    </div>
+    <div class="container containerSmall">
+      <form
+        action="#"
+        method="get"
+        data-qa="small"
+      >
+        <label for="smallSearch">
+          <img
+            src="./images/Search.svg"
+            class="smallIcon"
+            alt="icon"
+          >
+        </label>
+        <input
+        id="smallSearch"
+        type="search"
+        class="search smallSearch fokyc"
+        placeholder="Try &quot;Los Angeles&quot;"
```

Comment:
Add indents. Check 4th point from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)